Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -71,7 +71,7 @@ public OdsStartupBase(IWebHostEnvironment env, IConfiguration configuration)
 
             Configuration.Bind("Plugin", Plugin);
 
-            _logger.Debug($"built configuration = {Configuration}");
+            _logger.Info($"Configuration:{Environment.NewLine}{Configuration.GetDebugView()}");
```

Comment:
I'm concerned from a security perspective outputting this kind of configuration information, in particular doing so at an Info level.  This risks writing passwords or other sensitive information out to whatever loggers are set up, at a level that most people wouldn't expect.  Since this is just a logging improvement let's hold on this and discuss more post release.